pkg/openid4ci/openid4ci.go

```diff
@@ -8,17 +8,20 @@ SPDX-License-Identifier: Apache-2.0
 package openid4ci
 
 import (
+	"bytes"
+	"encoding/json"
 	"errors"
+	"fmt"
+	"io"
+	"net/http"
 	"net/url"
 	"strconv"
-
-	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
 )
 
 // NewInteraction creates a new OpenID4CI Interaction.
```

Comment:
rename input of this function to `initiateIssuanceURI`.

`requestURI` is term used for other entity in the request, it will lead to confusion.

Reply:
Done